l3kernel/l3tl.dtx

```diff
@@ -392,7 +392,7 @@
 %     \tl_if_regex_match:nnTF { example } { [b-dq-w] } { TRUE } { FALSE }
 %   \end{verbatim}
 %   leaves \texttt{TRUE} then \texttt{FALSE} in the input stream.
-%   Theses are alternative names for \cs{regex_match:nnTF} and friends,
+%   Theses are alternative names for \cs{regex_if_match:nnTF} and friends,
```

Comment:
Probably should start: 
These are ...

As no one is writing theses on this subject!